science/stimfit/Portfile

```diff
@@ -35,11 +34,11 @@ configure.args      --disable-dependency-tracking \
                     --with-biosiglite \
                     --with-wx-config=${wxWidgets.wxconfig}
 
-if {![variant_isset python39] && ![variant_isset python310]} {
-    default_variants +python310
+if {![variant_isset python39] && ![variant_isset python310] && ![variant_isset python311] && ![variant_isset python312]} {
+    default_variants +python312
```

Comment:
MacPorts' default version is 3.13 now, so if the port supports that it should be used.

Reply:
wxPython is not available for Python 3.13 so that stimfit cannot support 3.13.